scapy/arch/pcapdnet.py

```diff
@@ -101,21 +80,24 @@ def get_if_raw_addr(iff):
     try:
       p = devs
       while p:
-        if p.contents.name.endswith(iff.guid.encode("ascii")):
           a = p.contents.addresses
           while a:
             if a.contents.addr.contents.sa_family == socket.AF_INET:
               ap = a.contents.addr
               val = cast(ap, POINTER(sockaddr_in))
-              ret = b"".join(chb(x) for x in val.contents.sin_addr[:4])
+              conf.cache_ipaddrs[plain_str(p.contents.name)] = b"".join(chb(x) for x in val.contents.sin_addr[:4])
```

Comment:
Is this line longer than 80 characters ?

This comment applies to the other changes.

Reply:
What I'm going to do is force PEP8 on the entire pcapdnet file. It's the only one in scapy with 2 spaces indent and is absolutely horrible to read. I'll add a reformat PR after this one